model/clusters_mgmt/v1/flavour_type.model

```diff
@@ -19,13 +19,18 @@ limitations under the License.
 class Flavour {
 	// Default _Amazon Web Services_ settings of the cluster.
 	//
-	// These can be overriden specifying in the clsuter itself a different set of settings.
+	// These can be overridden specifying in the cluster itself a different set of settings.
```

Comment:
not all can be overridden. only the compute instance type

Reply:
Fixed